website/src/middlewares/api-v3/index.js

```diff
@@ -1,10 +1,8 @@
-'use strict';
-
 // This module is only used to attach middlewares to the express app
 
-let errorHandler = require('./errorHandler');
+import errorHandler from './errorHandler';
 
-module.exports = function attachMiddlewares (app) {
+export default function middleware (app) {
```

Comment:
I would keep the attachMiddlewares name, this isn't a proper middleware but a function that takes the app and attaches several middlewares to it